packages/integration-tests/src/utils.ts

```diff
@@ -127,3 +129,7 @@ export const dmodal = () => `div[aria-modal=true]`;
 export const generateTestName = () => `test_${generateStandardId(4)}`;
 
 export const randomString = () => crypto.randomBytes(8).toString('hex');
+
+export const devFeatureIt = isDevFeaturesEnabled ? it : it.skip;
+
+export const devFeatureDescribe = isDevFeaturesEnabled ? describe : describe.skip;
```

Comment:
nit: what about exporting an object like `devFeatureTest = { it, describe }`

Reply:
cool